story/src/0-Button.stories.tsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react';
 import { Button } from '../../frontend/src/component/ui/Button';
 import { action } from '@storybook/addon-actions';
-import { text, withKnobs } from '@storybook/addon-knobs';
-import { withInfo } from '@storybook/addon-info';
+import { text } from '@storybook/addon-knobs';
 
 import styles from './0-Button.stories.module.scss';
 
 export default {
   title: 'UI/Button',
-  component: <Button />,
-  decorators: [withKnobs, withInfo({ header: false, inline: true })]
```

Comment:
Is the doc style still looking correct?

Reply:
Yes, my understanding is that with Storybook Docs, even though it's built to support the new MDX format, you can still use Component Story Format (CSF).

I did notice that it seems as though you have to pass in the component type rather than a JSX element into the `component` field now in order to get the knobs to work. This is how they seem to do it [here](https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/docspage.md) too.